CMakeLists.txt

```diff
@@ -112,9 +112,10 @@ if(PYTHON_BINDINGS)
       set(PYTHON_LIBRARIES "$ENV{PYTHON_LIBRARIES}")
     endif()
     if(NOT PYTHON_INCLUDE_DIRS AND NOT PYTHON_LIBRARIES)
-        find_package(PythonLibs 2.7 REQUIRED)
-      else()
+      find_package(PythonLibs 2.7 REQUIRED)
+      if(NOT PYTHON_INCLUDE_DIRS AND NOT PYTHON_LIBRARIES)
```

Comment:
If set PYTHON_INCLUDE_DIRS and PYTHON_LIBRARIES as the params of CMake, line 117 would raise an exception.